gui/controlpanel/public/javascripts/client.js

```diff
@@ -68,15 +68,15 @@ function updateATOSStatusIcon(status, new_text) {
   }
 
 // HTML-Button callbacks
-function sendResetTest(){
-    sendCommand("send_reset_test", ws);
-}
 function sendInit(){
     sendCommand("send_init", ws);
 }
 function sendConnect(){
     sendCommand("send_connect", ws);
 }
+function sendDisconnect(){
```

Comment:
Hmm, I don't experience this. Could it be in combination with something else?

Reply:
It happen if I press `Init` -> `Connect` -> `Disconnect`, and I'm using one object. Maybe we can take a look at this together, because it seems to happen everytime for me.